docs/siem/detections/api/lists/api-update-list-container.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[lists-api-update-container]]
+=== Update list container
+
+Updates an existing list container.
+
+You can use `PUT` or `PATCH` methods to update list containers, where:
+
+* `PUT` replaces the original container and deletes fields that are not
+specified.
+* `PATCH` updates the specified fields.
+
+==== Request URL
+
+`PUT <kibana host>:<port>/api/lists`
+
+`PATCH <kibana host>:<port>/api/lists`
+
+==== Request body
+
+A JSON object with:
+
+* The `id` of the list container you want to update.
+* The fields you want to modify.
+
+IMPORTANT: If you call `PUT` to update a rule, all unspecified fields are
+deleted. You cannot modify the `id` and `type` fields.
+
+For `PATCH` calls, any of the fields can be modified, whereas for `PUT` calls,
+some fields are required. 
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required (`PUT` calls)
+
+|`description` |String |Describes the list container. |Yes
+|`meta` |Object |Placeholder for metadata about the list container. |No
+|`name` |String |The list container's name. |Yes
+
+|==============================================
+
+
+===== Example request
+
+Updates the `name` field:
+
+[source,console]
+--------------------------------------------------
+PATCH api/lists
+{
+  "id": "internal-ip-excludes",
+  "name": "Internal IP addresses"
+}
```

Comment:
As mentioned above, you can optionally put in a `_version` if you want OCC and have a value from the list with a GET call.